driver/network_driver.go

```diff
@@ -168,6 +178,15 @@ func (d NetworkDriver) CreateNetwork(request *network.CreateNetworkRequest) erro
 		}
 	}
 
+	for _, ipData := range request.IPv6Data {
+		// Don't support older versions of Docker which have a bug where the correct AddressSpace isn't provided
```

Comment:
The meaning of this comment isn't going to stand up over time. Can you include some more details, like which versions of Docker are supported and which aren't. If someone uses an old version of docker, will they just see an error saying "Non-Calico IPAM driver is used"? Can we do better than that?